### PR TITLE
issue: 824880 Fix gtest compilation issue for gcc 4.4.7

### PR DESCRIPTION
**Pull request by igor-ivanov**

### CODE REVIEW

**Comment by OphirMunk** [comment]
@igor-ivanov is there a squash needed?

**Comment by mellanox-github** [comment]
Test PASSed.
See http://bgate.mellanox.com/jenkins/job/accl-libvma-pr/550/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/accl-libvma-pr/557/ for details (Mellanox internal link).

**Comment by igor-ivanov** [comment]
@OphirMunk I see them separated (different issues) .

File: contrib/jenkins_tests/compiler.sh [hunk 1]
```diff
@@ -9,7 +9,7 @@ if [ $(command -v module >/dev/null 2>&1 || echo $?) ]; then
 	echo "[SKIP] module tool does not exist"
 	exit 0
 fi
-module load intel/ics
+module load intel/ics-15.0.1
```
Comment (OphirMunk) [inline contrib/jenkins_tests/compiler.sh]: @igor-ivanov please explain the motivation for this enforcement

**Comment by igor-ivanov** [comment]
Intel Compiler v16 reports compilation issues on VMA code. It makes all current PRs failed. So we should  resolve these issues in separate pull request and enable Intel Compiler v16 after this.

**Pull request merged**